### PR TITLE
Add doc for EnumToIntConverter

### PR DESCRIPTION
**Pull request by CliffAgius**
Changed the file name to match others in the project and fixed git issue locally...

### CODE REVIEW

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by CliffAgius** [comment]
Sorry team I somehow made a complete mess of my local git for the project and it was for the sake of one file far easier to just create a new PR I hope that is Ok.

File: docs/maui/converters/enum-to-int-converter.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    {
+      Picker picker = new Picker { Title="EnumToIntConverter"};
+      picker.SetBinding(Picker.ItemsSourceProperty, nameof(ViewModel.AllStates));
+      picker.SetBinding(Picker.SelectedItemProperty, nameof(ViewModel.SelectedState));
+
+      Content = new StackLayout
+			{
+				Margin = new Thickness(20),
+				Children = {
+					new Label { Text = "The EnumToIntConverter is a converter that allows users to convert a standard enum (extending int) to its underlying primitive int type.", FontAttributes = FontAttributes.Bold, HorizontalOptions = LayoutOptions.Center },
```
Comment (jfversluis) [inline docs/maui/converters/enum-to-int-converter.md:66]:
```suggestion
					new Label { Text = "The EnumToIntConverter is a converter that allows users to convert a standard enum (extending int) to its underlying primitive int type.",
					FontAttributes = FontAttributes.Bold,
					HorizontalOptions = LayoutOptions.Center },
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
Couple of small things here

**Comment by PRMerger16** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by CliffAgius** [comment]
Thanks @jfversluis for the help.

Still trying to learn the C# Markup stuff and great catch on the typo, I reviewed it myself a number of times and still missed it.

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger-2** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by jfversluis** [comment]
> Thanks @jfversluis for the help.
> 
> Still trying to learn the C# Markup stuff and great catch on the typo, I reviewed it myself a number of times and still missed it.

haha no worries, that's what the reviews are for!

File: docs/maui/converters/enum-to-int-converter.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      picker.SetBinding(Picker.ItemsSourceProperty, nameof(ViewModel.AllStates));
+      picker.SetBinding(Picker.SelectedItemProperty, nameof(ViewModel.SelectedState));
+
+      Content = new StackLayout
+			{
+				Margin = new Thickness(20),
+				Children = {
+					new Label { Text = "The EnumToIntConverter is a converter that allows users to convert a standard enum (extending int) to its underlying primitive int type.",
+					FontAttributes = FontAttributes.Bold,
+					HorizontalOptions = LayoutOptions.Center },
```
Comment (jfversluis) [inline docs/maui/converters/enum-to-int-converter.md:68]:
```suggestion
					HorizontalOptions = LayoutOptions.Center 
				},
```

File: docs/maui/converters/enum-to-int-converter.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    {
+      Picker picker = new Picker { Title="EnumToIntConverter"};
+      picker.SetBinding(Picker.ItemsSourceProperty, nameof(ViewModel.AllStates));
+      picker.SetBinding(Picker.SelectedItemProperty, nameof(ViewModel.SelectedState));
+
+      Content = new StackLayout
+			{
+				Margin = new Thickness(20),
+				Children = {
+					new Label { Text = "The EnumToIntConverter is a converter that allows users to convert a standard enum (extending int) to its underlying primitive int type.",
```
Comment (jfversluis) [inline docs/maui/converters/enum-to-int-converter.md:66]:
```suggestion
					new Label {
						Text = "The EnumToIntConverter is a converter that allows users to convert a standard enum (extending int) to its underlying primitive int type.",
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
Sorry couple more I just noticed!

You might want to checkout the "Add suggestion to batch" button btw so you can commit all these suggestions in one commit. If I can offer some unsolicited advice...

**Comment by PRMerger-2** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by bijington** [comment]
> Sorry team I somehow made a complete mess of my local git for the project and it was for the sake of one file far easier to just create a new PR I hope that is Ok.
> 
> 

Don't worry about that I do it all the time 😉

File: docs/maui/converters/enum-to-int-converter.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+      Content = new StackLayout
+			{
+				Margin = new Thickness(20),
+				Children = {
+					new Label {
+						Text = "The EnumToIntConverter is a converter that allows users to convert a standard enum (extending int) to its underlying primitive int type.",
+					FontAttributes = FontAttributes.Bold,
+					HorizontalOptions = LayoutOptions.Center 
+				},
```
Comment (jfversluis) [inline docs/maui/converters/enum-to-int-converter.md:70]:
```suggestion
						FontAttributes = FontAttributes.Bold,
						HorizontalOptions = LayoutOptions.Center 
					},
```

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by jfversluis** [comment]
Sorry for all the notification here people 😬 Thanks so much @CliffAgius !

**Review by jfversluis — APPROVED** [review]

**Pull request merged**